src/components/ItaliaTheme/View/Commons/LocationItem.jsx

```diff
@@ -31,50 +35,34 @@ const LocationItem = ({
   location,
   show_icon,
   show_title_link,
-  load = true,
   details_link = true,
 }) => {
   const intl = useIntl();
-  const url = flattenToAppURL(location['@id']);
-  const key = `luogo${url}`;
-  const locationContent = useSelector((state) => state.content.subrequests);
-  const loaded =
-    locationContent?.[key]?.loading || locationContent?.[key]?.loaded;
-  const dispatch = useDispatch();
-
-  useEffect(() => {
-    if (load && !loaded) {
-      dispatch(getContent(url, null, key));
```

Comment:
veniva fatta la load solo per avere l'immagine? o mancavano altre info che poi vengono mostrate sotto?

Reply:
`location` ha già tutto quello che serve, incluso `quartiere` e `circoscrizione`

Reply:
sempre? in qualsiasi punto viene usato il componente LocationItem? se si allora ok